# First and second order formation statistics

std_msgs/Header header

FormationStatistics stats